doc/filters/replace.rst

```diff
@@ -6,12 +6,12 @@ format is free-form):
 
 .. code-block:: twig
 
-    {{ "I like %this% and %that%."|replace({'%this%': fruit, '%that%': "oranges"}) }}
+    {{ "I like %this% and %that%."|replace({'%this%': apples, '%that%': "oranges"}) }}
```

Comment:
Replacing `fruit` with `apples` here is wrong. As the comment below indicates the example assumes that there is a variable called `fruit` whose value is `apples`.